components/CollectiveCard.js

```diff
@@ -283,10 +283,8 @@ class CollectiveCard extends React.Component {
                   <LabelWrapper>
                     <FormattedMessage
                       id="collective.card.collectives.count"
-                      defaultMessage="Hosting {n, plural, one {Collective} other {Collectives}}"
-                      values={{
-                        n: get(collective, 'stats.collectives.hosted'),
-                      }}
+                      defaultMessage="Hosted {n, plural, one {Collective} other {Collectives}}"
```

Comment:
Issue introduced in https://github.com/opencollective/opencollective-frontend/pull/5834

![image](https://user-images.githubusercontent.com/1556356/155282241-913ba17c-9806-4e2a-8dee-e1c463d6ba56.png)